modules/nw-exposing-service.adoc

```diff
@@ -46,18 +46,17 @@ route "mysql-80-rhel7" exposed
 ----
 
 . Use a tool, such as cURL, to make sure you can reach the service using the
-cluster IP address for the service:
+cluster IP address for the service. First, port-forward the service:
 +
 [source,terminal]
 ----
-$ curl <pod_ip>:<port>
+$ oc port-forward service/mysql-80-rhel7 <port>:3306
```

Comment:
You shouldn't need to do this (if the service is exposed via ingress routes). You only need to do this, if your not exposing a service. 

It would be better (since the example is mysql) that you use a mysql command (to connect from a local [mysql client](https://linux.die.net/man/1/mysql) to a the `--port` option (set it to 443) due to this being exposed with a TLS-passthough route). 

Or alter the service in the example to be a webserver or something else.